modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurators/RootProjectConfigurator.java

```diff
@@ -102,10 +99,14 @@ public RootProjectConfigurator(Settings settings) {
 
 	@Override
 	public void apply(Project project) {
-		WorkspaceExtension workspaceExtension = GradleUtil.getExtension(
+		final WorkspaceExtension workspaceExtension = GradleUtil.getExtension(
 			(ExtensionAware)project.getGradle(), WorkspaceExtension.class);
 
+		final AppDockerExtension appDockerExtension = GradleUtil.addExtension(
```

Comment:
Don't add the extension, get it with `GradleUtil.getExtension(project, AppDockerExtension.class)`.